docs/index.md

```diff
@@ -49,14 +49,13 @@ Pelorus works by integrating with your application platforms, developer tools, a
 
 <div class="flex cards" markdown>
 
--   # :simple-kubernetes:
 -   # :simple-redhatopenshift:
 -   # :simple-github:
--   # :simple-gitlab:
 -   # :simple-bitbucket:
+-   # :simple-gitea:
+-   # :simple-gitlab:
+-   # :simple-azuredevops:
 -   # :simple-jirasoftware:
--   # :fontawesome-brands-servicestack:
--   # :simple-trello:
```

Comment:
we are still missing service now icon (not available in fontawesome or the other)